Setup.bat

```diff
@@ -99,6 +100,9 @@ call :MarkStartOfBlock "Retrieve dependencies"
     spatial package retrieve worker_sdk      c-dynamic-x86_64-vc140_md-win32            %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-vc140_md-win32.zip"
     spatial package retrieve worker_sdk      c-dynamic-x86_64-gcc510-linux              %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-gcc510-linux.zip"
     spatial package retrieve worker_sdk      c-static-fullylinked-arm-clang-ios         %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c-static-fullylinked-arm-clang-ios.zip"
+    spatial package retrieve worker_sdk      c-dynamic-arm64v8a-clang_ndk16b-android    %PINNED_CORE_SDK_VERSION%   %DOMAIN_ENVIRONMENT_VAR%   "%CORE_SDK_DIR%\worker_sdk\c-dynamic-arm64v8a-clang_ndk16b-android.zip"

```

Comment:
Any idea how big these packages are? Do we want to default include them?

Reply:
Just checked. They are all between 5-6 MB big.  if you prefer, we can have a mobile flag or something similar that will download it if needed. This would probably be useful for the iOS package that is pretty big and has caused timeouts before.

Reply:
Flag sounds sensible. cc: @ernopp

Reply:
added a --mobile flag for both Setup scripts.